app/models/chef.rb

```diff
@@ -1,4 +1,9 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :ingredients, through: :dishes
+
+   def unique_ingredients_list
+      ingredients.distinct
```

Comment:
Great work! Adding the additional association to ingredients help make this logic a lot easier.